docs/building-rnw.md

```diff
@@ -1,77 +1,67 @@
----
-id: building-rnw
-title: Building React-Native-Windows
----

```

Comment:
@chiaramooney these changes seem wrong (e.g. removing the docusaurus headers), can you clarify where the changes are coming from?

Reply:
Hmm I grabbed the doc changes from [0.65-stable/docs](https://github.com/microsoft/react-native-windows/tree/0.65-stable/docs)